### ISSUE TITLE
increase client_header_buffer_size

### ISSUE THREAD

**Issue by walderT** [codekitchen/dinghy-http-proxy#31]
We are using the webdevops/vagrant-docker-vm (https://github.com/webdevops/vagrant-docker-vm). In this VM the dinghy-http-proxy is in usage.
My Problem: 
I get an "upstream sent too big header while reading response header from upstream,” Error.
I think its related to an xdebug Bug which sends to many headers, in my case the header
Set-Cookie: XDEBUG_SESSION=docker; expires=Wed, 12-Apr-2017 08:02:41 GMT; Max-Age=3600; path=/
is 140 times send.
There ist already an Ticket at: https://bugs.xdebug.org/view.php?id=1397

Nobody knows when this will be resolved.

My question, is it possible that you increase the “client_header_buffer_size” as workaround for this issue?

Thank you

**Comment by codekitchen** [comment]
Heh fun. When using the proxy with Dinghy, you can [add your own custom nginx configuration](https://github.com/codekitchen/dinghy#http-proxy), but this is the first I've heard of the vagrant-docker-vm project so I don't know if it has any similar functionality.

It seems fine to just always set `client_header_buffer_size` in https://github.com/codekitchen/dinghy-http-proxy/blob/master/dinghy.nginx.conf , do you want to figure out a reasonable value and send over a PR?

**Comment by mblaschke** [comment]
I'm maintainer of the vagrant docker vm project.
This issues it not really related to the vm (ubuntu 16.04 vm with dory) but it seems to be a xdebug issue inside a container.

**Comment by codekitchen** [comment]
Yeah that's how I read the original report as well, he linked to the relevant xdebug issue
